docs-chef-io/content/workstation/getting_started.md

```diff
@@ -19,60 +19,37 @@ new Chef Workstation installation. If you do not have Chef Workstation
-
-    ``` bash
-    echo 'eval "$(chef shell-init bash)"' >> ~/.bash_profile
-    ```
+A good visual code editor is not a requirement for working with Chef
+Infra, but a good code editor can save you time. A code editor should
+support the following: themes, plugins, snippets, syntax Ruby code
+coloring/highlighting, multiple cursors, a tree view of the entire
+folder/repository you are working with, and a Git integration.
 
-    where `bash` and `~/.bash_profile` represents the name of the shell.
+These are a few common editors:
 
-    If zsh is your preferred shell then run the following:
+- [Visual Studio Code (free/open source)](http://code.visualstudio.com)
+- [GitHub Atom - (free/open source)](http://atom.io)
 
-    ``` bash
-    echo 'eval "$(chef shell-init zsh)"' >> ~/.zshrc
-    ```
+Chef Infra support in editors:
 
-3. Run `which ruby` again. It should return
-    `/opt/chef-workstation/embedded/bin/ruby`.
+- [VSCode Chef Infra Extension](https://marketplace.visualstudio.com/items?itemName=chef-software.Chef)
+- [Chef on Atom](https://atom.io/packages/language-chef)
+## Set up Ruby
 
-{{< note >}}
+While you don't have to know the Ruby programming language to use Chef, you do need to make sure that your computer knows where to find Ruby in order to run.
 
-Using Chef Workstation-provided Ruby as your system Ruby is optional.
-For many users, Ruby is primarily used for authoring Chef cookbooks. If
-that's true for you, then using the Chef Workstation-provided Ruby is
-recommended.
+* If you don't plan on writing a lot of Chef cookbooks and you don't use Ruby for other purposes, then set up Ruby with an [environment variable]({{< ref "#Add Ruby as an Environment Variable" >}})
+* If you plan on using Ruby for authoring Chef cookbooks but don't use it for other purposes, then we recommend that you use [Chef Workstation's Ruby as the system Ruby]({{< ref "#Use Chef Workstation Ruby as the System Ruby">}}) Use Chef Workstation Ruby as the System Rubyusing the version of Ruby provided in Chef Workstation. Using Chef Workstation's Ruby as your system Ruby is optional.
+* If you use enough different versions of Ruby that you use a Ruby version manager such as RVM or rbenv, then make sure that you have the same version of Ruby available as we ship with Chef Workstation. You probably already know how to do it, but `rvm --help` or `rbenv --help` are great places to start.
 
-{{< /note >}}
+### Add Ruby as an Environment Variable
 
-Add Ruby to \$PATH
-==================
+Chef Workstation includes a stable version of Ruby as part of its installer.
+The path to this version of Ruby must be added to the `$PATH` environment variable and saved in the configuration file for the command shell (Bash, csh, and so on) that is used on the machine running Chef Workstation.
 
-Chef Infra Client includes a stable version of Ruby as part of its
```

Comment:
I like how we talked about infra client here even though we're in workstation. 🤦